test/Ogmios/Aeson.purs

```diff
@@ -127,16 +127,19 @@ loadFixtures = do
     catMaybes <$> flip parTraverse (ourFixtures <> ogmiosFixtures) \fp -> do
       let bn = basename fp
       contents <- readTextFile UTF8 fp
-      aeson <- liftEither $ lmap
-        (error <<< ((bn <> "\n  ") <> _) <<< printJsonDecodeError)
-        (Aeson.parseJsonStringToAeson contents)
-      pure case pattern >>= flip match bn >>> map tail of
-        Just [ Just query ] -> Just
-          { query
-          , bn
-          , aeson
-          }
-        _ -> Nothing
+      -- ignore empty (corrupted) fixtures
```

Comment:
Are there any? Where are they from? can we get rid of them?

Reply:
Yes, there is at least one in the ogmios repository